auth/example/roles2.yaml

```diff
@@ -0,0 +1,51 @@
+roles:
+  - name: "agent"
+    rules:
+      - key:
+          type: "action"
+          namespace: "default"
+        methods:
+          - "set"
+          - "delete"
+      - key:
+          type: world
+          name: "judge"
+          namespace: "default"
+        methods:
+          - "get"
+          - "find"
+          - "watch"
+  - name: "judge"
+    rules:
+      - key:
+          type: "action"
+        methods:
+          - "get"
+          - "watch"
+      - key:
+          type: "world"
+        methods:
+          - "get"
+          - "set"
+          - "find"
+          - "watch"
+          - "delete"
+  - name: "logger"
+    rules:
+      - key:
```

Comment:
Didn't find expected key

Reply:
I haven't bring this case in my test cases. What exactly do you mean?
I also didn't altered test files. I just copied them from files config and example.